packages/manager/apps/pci-savings-plan/src/utils/formatter/date.ts

```diff
@@ -25,7 +25,8 @@ export const getLastXMonths = (
     year: 'numeric',
   });
   for (let i = 0; i < nbrMonths; i += 1) {
-    months[i] = formatter.format(date);
+    const formattedDate = formatter.format(date);
+    months[i] = formattedDate.charAt(0).toUpperCase() + formattedDate.slice(1);
```

Comment:
Upper case should be handled by the visual layer when possible https://tailwindcss.com/docs/text-transform#uppercasing-text